compiler/src/iree/compiler/Codegen/Common/TileAndDistributeToWorkgroupsPass.cpp

```diff
@@ -237,6 +239,8 @@ struct LowerDispatchWorkgroupCountForDagRootOp
 
   /// Loops that are partitioned.
   SmallVector<unsigned> partitionedLoops;
+
+  int32_t maxWorkgroupParallelDims;
```

Comment:
Please add comments.

Reply:
I removed this variable.

Reply:
The variable is still here.....

Reply:
eeh, yeah not sure what happened in my brain. pinky swear its commented this time.